INSTALL-STANDALONE.md

```diff
@@ -24,59 +20,36 @@ The installation steps below will allow you to set-up the application as a stand
 5. Tap _"Always allow from this computer"_.
 6. Tap **OK**.
 
-## 2. Building the onecourse app
+## 2. Building the Pehla Class app
 
 1. Ensure you have **Android Studio** installed. To do this, you can [follow the steps in this helpful guide](https://developer.android.com/studio/install.html).
+     
+2. Start Android Studio and import the project into it.
 
-2. Configure the _ROOT_FOLDER_ environment variable and _ANDROID_HOME_ environment variable based on the location of the Android SDK:  
-   
-   Follow these steps for macOS (Mavericks or later) or Linux-based system:
-	
-		export ROOT_FOLDER=onecourse
-		
-		export ANDROID_HOME=[PATH/TO/ANDROID_SDK]
-		
-   Follow these steps for a Windows 7 or later system:
+3. To change the app locale, just select the desired language from the build variant.
 
-		Set-Variable -Name "ROOT_FOLDER" -Value "onecourse"
-		
-		Set-Variable -Name "ANDROID_HOME" -Value "PATH/TO/ANDROID_SDK"
-		
-3. Clone the **onecourse source** repository:
-		
-		git clone https://github.com/XPRIZE/GLEXP-Team-onebillion.git $ROOT_FOLDER/
-		
-4. In the _ROOT_FOLDER_ build _gradle_ tasks:
+4. Add google-services.json file in language specific [app](https://github.com/maqsoftware/Pehla-Class/tree/master/app/src) folder. It is there in Firebase console after the app project has been created.
 
-        ./gradlew tasks
+5. Build the project.
 		
-5. Build the **onecourse** _.apk_:
-
-	onecourse English:
-
-          ./gradlew assembleEnGB_community_debug
-	  
-	onecourse Swahili:
-	
-          ./gradlew assembleSw_community_debug
-		  
-6. Download the language-specific assets from the [releases](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/tag/v3.0.0) section and extract into your _ROOT_FOLDER_.
-		
-## 3. Installing onecourse
+## 3. Installing Pehla Class
 
 1. Install the application via adb:
 
-	onecourse English:
+	Pehla Class English:
 
-		adb install app/build/outputs/apk/enGB_community_/debug/app-enGB_community_-debug.apk
+		adb install app/english/release/app-english-release.apk
```

Comment:
I think the users would select the debug version by default. Hence, we should refer to the debug version folder and APK.